### PR TITLE
#225405: refactored override styles

### PR DESCRIPTION
**Pull request by dubunin**
The task here was to make sure the fabric-react version matched up with the fabric 3.0 version. Everything looked good but there were still some fabric-react specific overrides that I went ahead and refactored.

### CODE REVIEW

**Comment by Jahnp** [comment]
Approved

**Comment by dzearing** [comment]
Doesn't this break links that look like buttons?

File: src/components/Link/Link.scss [hunk 1]
```diff
@@ -39,22 +44,9 @@
   @include ms-Link;
 }
 
-// TODO: Remove overrides below.
```
Comment (dzearing) [inline src/components/Link/Link.scss]: It seems like you're removing all the content in BUTTON.ms-Link ?

Won't that break buttons that use ms-Link styling?

Comment (Jahnp) [inline src/components/Link/Link.scss]: Ah, good catch--yes, that would definitely break that scenario. @dubunin--why is this being removed?

Comment (dubunin) [inline src/components/Link/Link.scss]: The only thing I left was the border: none; on the BUTTON.ms-link, however I just tried moving that up to the ms-link mixin as well and it works fine. Just added an update with the entire BUTTON.ms-link selector removed as it was not necessary. See screenshot below. The link on the right is the button:

after

**Pull request merged**